.github/workflows/publish-openapi.yaml

```diff
@@ -0,0 +1,30 @@
+name: OpenAPI Publisher
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - openapi/**
+
+jobs:
+  publish:
+    workflow_run:
+      workflows: ['OpenAPI Validator']
+      branches: [main]
+      types:
+        - completed
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        name: Check out repository
+
+      - uses: readmeio/rdme@7.2.0
+        name: Sync Open Payments API spec
+        with:
+          rdme: openapi openapi/RS/openapi.yaml --key=${{ secrets.README_API_KEY }} --id=${{ secrets.README_OP_API_ID }}
+
+      - uses: readmeio/rdme@7.2.0
+        name: Sync Auth Server API spec
+        with:
+          rdme: openapi openapi/AS/client/openapi.yaml --key=${{ secrets.README_API_KEY }} --id=${{ secrets.README_AUTH_API_ID }}
```

Comment:
Should AS <> RS be published?

Reply:
We need to create the placeholder on readme.io first. Should I just merge it as is now and then we decide which ones to move out and which ones to publish to readme or should we decide before I merge?

Reply:
ship it :rocket: 
(plus I'm convinced by Adrian to remove AS <> RS, so it shouldn't be published)